web/modules/custom/bnf/bnf_client/bnf_client.install

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the bnf module.
+ */
+
+/**
+ * Add the last field to subscriptions.
+ */
+function bnf_client_update_10101(): void {
+  $entity_type = \Drupal::service('entity_type.manager')->getDefinition('bnf_subscription');
+  \Drupal::entityDefinitionUpdateManager()->updateEntityType($entity_type);
+}
+
+/**
+ * Trigger job_scheduler rebuild.
+ */
+function bnf_client_update_10102(): void {
+  // Rebuild scheduler info so it notices our new periodic job.
+  job_scheduler_rebuild_all();
```

Comment:
Is this something that needs to be added to `hook_install` also?

Reply:
You're right. `job_scheduler` apparently only ever refreshes hook info when saving the settings form. An odd choice considering its main target is developers.